src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +17,15 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .flatMap(x -> Arrays.stream(x.split(",")))
+               .map(Integer::parseInt)
+               .filter(x -> x % 2 == 0)
+                .min(Integer::compareTo)
```

Comment:
```suggestion
                .min()
```
Default behavior. Also, fix indents

Reply:
I get an error if I remove